crates/biome_js_syntax/src/expr_ext.rs

```diff
@@ -211,6 +211,12 @@ impl JsBinaryOperator {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum BinaryExpressionNodePosition {
```

Comment:
The name is not very helpful. What is "node" in a binary expression? 

Unless you document it, it's hard to understand it. (I don't)